app/src/main/java/org/oppia/app/databinding/ImageViewBindingAdapters.java

```diff
@@ -149,4 +151,20 @@ public boolean onResourceReady(
       }
     }
   }
+
+  /**
+   *
+   */
+  @BindingAdapter("app:onClick")
+  public static void setOnClick(@NonNull ImageView imageView, View.OnClickListener listener) {
```

Comment:
Are this & the next binding adapters still needed now that the circle image library reference was fixed?

Reply:
No, they aren't being used at all - I will delete them.